src/testsuite/arith128_test_i128.c

```diff
@@ -4764,11 +4764,192 @@ test_setbq (void)
 }
 
 //#define __DEBUG_PRINT__ 1
+int
+test_cmpuq_p2 (void)
+{
+  vui128_t i1, i2, e;
+  vb128_t j;
+  int rc = 0;
+  printf ("test_cmpltuq\n");
+
+  i1 = CONST_VINT128_DW128(0, 0);
+  i2 = CONST_VINT128_DW128(0, 0);
+  e =  CONST_VINT128_DW128(0, 0);
+  j = vec_cmpltuq(i1, i2);
+
+#ifdef __DEBUG_PRINT__
+  print_vint128x ("vcmpltuq( ", i1);
+  print_vint128x ("         ,", i2);
+  print_vint128x ("        )=", j);
+#endif
+  rc += check_vb128c ("vec_cmpltuq:", (vb128_t)j, (vb128_t) e);
```

Comment:
`j` doesn't need the cast to it's type.  _this is a global comment_

Reply:
Well it was required before. Now it is correct but redundant. 
A global fine/replace would be dangerous as this construct is use often.  I propose putting this in the back log to be addressed later (when fewer parts are in flux).